dll/win32/shell32/CCopyAsPathMenu.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * PROJECT:     ReactOS shell32
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Copy as Path Menu implementation
+ * COPYRIGHT:   Copyright 2024 Whindmar Saksit <whindsaks@proton.me>
+ *              Copyright 2024 Thamatip Chitpong <thamatip.chitpong@reactos.org>
+ */
+
+#pragma once
+
+class CCopyAsPathMenu :
+    public CComCoClass<CCopyAsPathMenu, &CLSID_CopyAsPathMenu>,
+    public CComObjectRootEx<CComMultiThreadModelNoCS>,
+    public IContextMenu,
+    public IShellExtInit
+{
+private:
+    UINT m_idCmdFirst, m_idCmdAction;
+    CComPtr<IDataObject> m_pDataObject;
+
+    HRESULT DoCopyAsPath(IDataObject *pdto);
```

Comment:
Suggestion: As done for other shell classes, prefix private function members with an underscore:
```suggestion
    HRESULT _DoCopyAsPath(IDataObject *pdto);
```

Reply:
That's unnecessary. I found that only a few classes in shell32 main code folder do that.